install/kubernetes/cilium/values.yaml

```
@@ -40,7 +40,7 @@ k8sServiceHost: ""
 k8sServicePort: ""
 
 cluster:
-  # -- Name of the cluster. Only required for Cluster Mesh.
+  # -- Name of the cluster. Only required for Cluster Mesh and mTLS auth with spire.
```

Capitalize consistently.
```suggestion
  # -- Name of the cluster. Only required for Cluster Mesh and mTLS auth with SPIRE.
```

---

thanks :100: